### DISCUSSION TITLE
Report Design and Viewer version issue

### DISCUSSION THREAD

**Discussion by piovezanfernando** [eclipse-birt/birt#1021] [General]
Hi, I just created two reports using version 4.9 of design report the report is working perfectly running local, but when sending the report to the server I get the error

```There is no report design object available.```

To fix the problem I edit the rptdesign file and change the version of the report tag as in the image below.

![image](https://user-images.githubusercontent.com/38543704/181764349-ae0bbff6-5ed9-4de3-bf86-e5a6541f65a8.png)

If the report is with version `3.2.24` the error is displayed but changing to version `3.2.23` the problem is solved and the report is rendered normally.

This second print is from the server running the viewer

![image](https://user-images.githubusercontent.com/38543704/181764672-77a92526-5080-47b6-a71c-7285cacc91e5.png)

Is there anything that can be done to resolve this issue?

thanks

**Comment by hvbtup** [comment]
The version number was increased with #845. Ideally, the version number should have increased many years ago.
Probably your BIRT server was built before #845 was merged.

**Reply by piovezanfernando** [reply]
This was exactly the problem. Although my installation is new I used an older version in the viewer (4.9.0.v201905231911) of the one I was using in development (4.9.0-20220315). It would be nice if they added a minor version 4.9.1, it would already help to avoid this kind of confusion and also a more friendly error message for when it's a version error maybe it would be interesting to avoid future problems.

Thank you for your help

**Reply by claesrosell** [reply]
@fernandopiovezan1, be aware that 4.9.0.v201905231911 is not a released version of Birt. It is just an early snapshot of what later became 4.9.0. You should probably not use that build at all.
